lib/apiservers/service/restapi/handlers/vch_create_test.go

```diff
@@ -0,0 +1,296 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package handlers
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"reflect"
+	"sort"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	cli "gopkg.in/urfave/cli.v1"
+
+	"github.com/vmware/govmomi/list"
+	"github.com/vmware/govmomi/vim25/types"
+	"github.com/vmware/vic/cmd/vic-machine/common"
+	"github.com/vmware/vic/cmd/vic-machine/create"
+	"github.com/vmware/vic/lib/apiservers/service/models"
+	"github.com/vmware/vic/pkg/trace"
+)
+
+type mockFinder struct {
```

Comment:
Would it be valuable to make this embed a testify `mock.Mock` so that we can verify that the `Element` method is called with the expected arguments?

Reply:
Sure, I will look into this.

Reply:
Will add this at a later time as per our slack convo.